canonical/ubuntu.com
# Refresh /server page

*Please review [/server/hyperscale](https://github.com/canonical/ubuntu.com/pull/13521) first*

## Done

- Refreshes /server based on [design](https://www.figma.com/file/Kw2SwMkH49tN54NbnKs5dt/24.04-U.com---server-(rebranding)?node-id=0%3A1&mode=dev) and [copydoc](https://docs.google.com/document/d/1vxLhtbjgPs9PCNEhr7fbhEoErCPMXYFfU4UTKH7Kbq4/edit) changes
- Moves the footer into a template
- Fly by: Adds some spaces to style.scss

## QA

- Go to [the demo]()
- Check that it matches the [design](https://www.figma.com/file/Kw2SwMkH49tN54NbnKs5dt/24.04-U.com---server-(rebranding)?node-id=0%3A1&mode=dev) and the [copydoc](https://docs.google.com/document/d/1vxLhtbjgPs9PCNEhr7fbhEoErCPMXYFfU4UTKH7Kbq4/edit)
- Check the page on large, medium and small screens
- Check the code abides by [vanilla](https://vanillaframework.io/) best practices

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8325
